tests/unit_tests/test_public_api.py

```diff
@@ -212,6 +212,33 @@ def test_artifact_download_logger():
             termlog.assert_not_called()
 
 
+def test_update_aliases_on_artifact(user, relay_server, wandb_init):
+    project = "test"
+
+    with relay_server():
```

Comment:
The way the test is written now you don’t need the relay server, since the user fixture will already create a new user in the local server that your test is using to log the artifacts. You will need the really server only if you want to do injections or assert on the requests responses from the local server. In your case you could’ve replaced the public apis calls in theory if the responses for the relevant artifact gql queries have the info you need

Reply:
Nice, thanks for the additional context. I'll take out the relay call in the next commit